app/src/main/java/org/p2p/wallet/AppModule.kt

```diff
@@ -30,5 +54,37 @@ object AppModule {
                 tokenKeyProvider = get()
             )
         }
+
+        includes(
```

Comment:
I am not sure if it's a good idea. 
Would be nice to see all the modules in app class.
I think the app module is still about common dependencies only and there is no place for other modules

Reply:
it's really convenient, now we need to add new module only one place and then it would be automatically added into dependency graph unit test

Reply:
It was the only entry place before as well, nah?

Reply:
no, we had an list of modules in the unit tests too
and we could forget to add module there